routes/api.js

```diff
@@ -90,4 +93,35 @@ api.delete('/friendships/user/:secret', async function (req, res) {
     })
 })
 
+// Route for merging accounts
+api.put('/user/:secret', async function (req, res) {
```

Comment:
hmmm what if we want to use PUT `user/:secret` to edit the user's name and stuff? We should probably have a different route for merging